### PR TITLE
Remove code for unsupported django.VERSION < (3, 2)

### PR DESCRIPTION
**Pull request by cclauss**
~% `django-upgrade --target version=5.0 **/*.py`~

~https://github.com/adamchainz/django-upgrade/blob/main/README.rst~

--
```diff
- import django

- if django.VERSION < (3, 2):
-     default_app_config = 'django_celery_beat.apps.BeatConfig'
```

### CODE REVIEW

**Comment by bukacdan** [comment]
What's the status on this? We want to move to Django 5 and we want to keep celery-beat in.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ repos:
         args: ["--py37-plus"]
 
   - repo: https://github.com/PyCQA/flake8
-    rev: 6.1.0
+    rev: 7.0.0
```
[the remark this answers is not in the document]
Comment (cclauss) [reply .pre-commit-config.yaml:12]: If the tests pass, why not do it here and now?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -30,3 +30,9 @@ repos:
     rev: 5.13.2
     hooks:
       - id: isort
+
+  - repo: https://github.com/adamchainz/django-upgrade
+    rev: 1.15.0
+    hooks:
+      - id: django-upgrade
+        args: [--target-version, "5.0"] # Replace with Django version
```
[the remark this answers is not in the document]
Comment (cclauss) [reply .pre-commit-config.yaml:38]: This is not clear from the documentation of that tool.  It is also unclear if I target `5.0`, will all previous upgrades also be run?

I have had difficulty upgrading from one version to the next without django-upgrading to that new version.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -30,3 +30,9 @@ repos:
     rev: 5.13.2
     hooks:
       - id: isort
+
+  - repo: https://github.com/adamchainz/django-upgrade
+    rev: 1.15.0
```
[the remark this answers is not in the document]
Comment (cclauss) [reply .pre-commit-config.yaml:35]:
```suggestion
    rev: 1.16.0
```

File: .pre-commit-config.yaml (hunk 1 above)
Comment (cclauss) [inline .pre-commit-config.yaml:12]:
```suggestion
    rev: 6.1.0
```
This was already done in a separate pull request. ;-)

File: .pre-commit-config.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (cclauss) [reply .pre-commit-config.yaml:38]: Do any of the changes made below break Django 3.2 or 4.2?

[the remark this answers is not in the document]
Comment (ulgens) [reply .pre-commit-config.yaml:38]: > This is not clear from the documentation of that tool.

I don't think this should be expected from that tool. It's more about where and why it's being used. This could make perfect sense if there were only supported version, or we were 5.0 as the minimum supported version.

> It is also unclear if I target 5.0, will all previous upgrades also be run?

I think this is actually clear:

> Pass your Django version as <major>.<minor> to the --target-version flag. django-upgrade will run all its fixers for versions up to and including the target version. 

https://github.com/adamchainz/django-upgrade?tab=readme-ov-file#usage

[the remark this answers is not in the document]
Comment (ulgens) [reply .pre-commit-config.yaml]: The setup was nice except for the target version. I think it should be in the repo, but branch name is losing it's connection with the PR context by every review comments 🙁

**Comment by cclauss** [comment]
Closing in favor of #729
